.github/workflows/ci.yml

```diff
@@ -0,0 +1,117 @@
+name: rollbar-java CI
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    name: Java ${{ matrix.java }}
+    strategy:
+      matrix:
+        # TODO: Add 11 once we fix Mockito's issues
+        java: [8]
+    steps:
+      - uses: actions/checkout@v2
+      # Our build uses JDK7's rt.jar to make sure the artifact is fully
+      # compatible with Java 7, so we let this action set Java 7 up for us
+      # and we store its JAVA_HOME
+      - name: Set up Java 7
+        uses: actions/setup-java@v1
+        with:
+          java-version: 7
+
+      - name: Capture JDK7_HOME
+        run: echo "export JDK7_HOME=\"$JAVA_HOME\"" > ~/.jdk7_home
+
+      # This is the JDK that'll run the build
+      - name: Set up Java ${{ matrix.java }}
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java }}
+
+      - name: Cache Gradle packages
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-${{ matrix.java }}-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-${{ matrix.java }}-
+
+      - name: Environment info
+        run: |
+          ./gradlew --version
+          echo ANDROID_SDK_ROOT: ${ANDROID_SDK_ROOT}
```

Comment:
The following SDK versions and build tools are installed in the base image:

https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md#android

This includes compile and target SDK 27, and build tools 28.0.3, which is what we set in our `build.gradle`. 

It might still be a good idea to explicitly install them even if they are currently available in the base image, since GH has already removed SDK versions from the base image, and it could eventually remove a version we need. I'll see if we can do the install on the same dir the base image puts the SDKs, that way if the versions we need are already there, it won't add much time to the build.

Reply:
It turns out gradle will install any missing SDK, so that's our fallback if the base image drops one of the versions we need. No need to update the PR